### DISCUSSION TITLE
--use-cluster and submit.slurm.sh

### DISCUSSION THREAD

**Discussion by SamuelECooper** [HALFpipe/HALFpipe#504] [Q&A] [answered]
Hi all,

I am running HALFpipe on an HPC cluster with the `--use-cluster` flag and the sif from https://download.fmri.science/singularity/halfpipe-halfpipe-latest.sif. Everything goes fine in an interactive session, but once the interface exists it immediately goes into running fmriprep and there is no submit.slurm.sh scripted produced that I can use to submit this as a job. Any ideas? Thanks so much!

Best,
Sam Cooper

**Comment by HippocampusGirl** [comment]
Hi @SamuelECooper,

Could you send me the command that you use to start HALFpipe?

Thanks :-)

Best,
Lea

**Comment by SamuelECooper** [comment]
Yep, here you go @HippocampusGirl - thanks for the quick response!

`singularity run --containall --bind /:/ext halfpipe-halfpipe-latest.sif --use-cluster`

**Reply by HippocampusGirl** [reply]
Hi @SamuelECooper, That's odd, the same options are working for me. Could you also send me the "log.txt" file that was generated in your working directory? You can either upload it here or e-mail it to <enigma@charite.de> :-)

**Reply by SamuelECooper** [reply]
Sure! I've attached it here, thanks!

[log.txt](https://github.com/HALFpipe/HALFpipe/files/10167818/log.txt)

**Comment by HippocampusGirl** [comment, marked as answer]
Hi @SamuelECooper, 

From the log file it seems like HALFpipe was cancelled before it was able to generate the submission script. 

HALFpipe needs to check the metadata for all subjects to figure out which subjects actually need to be run in the cluster job (and which ones to skip because they are incomplete or not selected for processing). This usually takes a short while, and is nothing to worry about.

I would recommend trying again and letting it run a bit longer.

**Reply by SamuelECooper** [reply]
Whoops, OK, that indeed worked - I thought it was going right into fmriprep. Thanks so much for your help and glad it was only user error!

Sam
